### PR TITLE
Objects Automated Test - LPS-138813 | master

### PR DESCRIPTION
**Pull request by evanilson-santana**

### CODE REVIEW

**Comment by evanilson-santana** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/tests/tests/CreateObject.testcase [hunk 1]
```diff
@@ -941,12 +941,20 @@ definition {
...
+			labelName = "Custom Object",
+			objectName = "CustomObject",
+			pluralLabelName = "Custom Objects");
 
-		// TODO LPS-138813 CanViewObjectsDetailsByClickingEyeIcon pending implementation
+		ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject");
+
+		ObjectAdmin.openObjectAdmin();
+
+		CreateObject.viewObject(labelName = "CustomObject");
```
Comment (rodrigocunhaa) [inline modules/apps/object/object-test/src/testFunctional/tests/tests/CreateObject.testcase:955]: Hi @evanilson-santana ,

for this test, you need to view the Object details through the eye (view) option on the kebab menu,

You have to use the `ObjectAdmin.selectCustomObjectFieldKebabMenuOption` macro. If I'm not mistaken, this macro is broken, so you'll have to fix it too.

Thanks!

**Review by rodrigocunhaa — CHANGES_REQUESTED** [review]
Hi @evanilson-santana ,

Please see my comments and make the necessary changes.

Thanks!

**Comment by evanilson-santana** [comment]
Fixed.

**Comment by rodrigocunhaa** [comment]
ci:forward

**Pull request closed without merging**